lib/auth/types/AuthType.js

```diff
@@ -244,10 +253,10 @@ export default class AuthType {
         this.server.auth.strategy('security_access_control', 'security_access_control_scheme', this.getCookieConfig());
         this.server.auth.strategy('security_access_control_cookie', 'cookie', this.getCookieConfig());
 
-         this.server.auth.default({
+        this.server.auth.default({
             mode: 'required', // @todo Investigate best mode here
             strategy: 'security_access_control' // This seems to be the only way to apply the strategy to ALL routes, even those defined before we add the strategy.
-         });
+        });
```

Comment:
I assume no change before this apart from "routesToIgnore"?

Reply:
Yes